### PR TITLE
Added glow enchant

### PR DESCRIPTION
**Pull request by NCBPFluffyBear**
- Added Glow enchant (can be used in future as an activation or cosmetic enchant)
- Replaced NANO_BLADE_ACTIVE_ENCHANT with GLOW
- Applied glow on Advanced Circuit (Cosmetic)

### CODE REVIEW

File: src/main/java/dev/j3fftw/litexpansion/utils/Constants.java [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 public final class Constants {
 
-    public static final NamespacedKey NANO_BLADE_ACTIVE_ENCHANT = new NamespacedKey(LiteXpansion.getInstance(),
-        "nano_blade_active_enchant");
+    public static final NamespacedKey GLOW = new NamespacedKey(LiteXpansion.getInstance(),
```
Comment (WalshyDev) [inline src/main/java/dev/j3fftw/litexpansion/utils/Constants.java]: Name this GLOW_ENCHANT

**Review by WalshyDev — COMMENTED** [review]
Seems pretty good, just a little nit picky thing

File: src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java [hunk 2]
```diff
@@ -10,18 +10,17 @@
 import javax.annotation.Nonnull;
 import java.util.Optional;
 
-@SuppressWarnings("NullableProblems")
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:13]: why is this removed

File: src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java [hunk 3]
```diff
@@ -61,9 +60,11 @@ public boolean canEnchantItem(ItemStack item) {
             final ItemMeta itemMeta = item.getItemMeta();
             final Optional<String> id = SlimefunPlugin.getItemDataService().getItemData(itemMeta);
             if (id.isPresent()) {
-                return id.get().equals(Items.NANO_BLADE.getItemId());
+                return id.get().equals(Items.ADVANCED_CIRCUIT.getItemId())
+                        || id.get().equals(Items.NANO_BLADE.getItemId());
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:65]: isnt there a better way to do this @WalshyDev

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

File: src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java (hunk 3 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:65]: I was thinking that maybe i could put these in an array and then define glowable items at the top

Comment (J3fftw1) [reply src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:65]: it will look ugly really quick

Comment (WalshyDev) [reply src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:65]: Not really, at least, not a way that matters to do

Comment (WalshyDev) [reply src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:65]: Probably should be in an ImmutableSet or something but... can't do that in Bukkit IIRC and we don't run J9.

File: src/main/java/dev/j3fftw/litexpansion/Items.java [hunk 4]
```diff
@@ -326,5 +328,9 @@
         "&7Refined Smeltery"
     );
 
+    static {
+        ADVANCED_CIRCUIT.addUnsafeEnchantment(Enchantment.getByKey(Constants.GLOW_ENCHANT), 1);
```
Comment (WalshyDev) [inline src/main/java/dev/j3fftw/litexpansion/Items.java]: Any reason this is addUnsafe despite the fact max level is 1 anyway?

File: src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java [hunk 5]
```diff
@@ -46,6 +46,17 @@ public void onEnable() {
         getServer().getPluginManager().registerEvents(new Events(), this);
 
         // Category
```
Comment (WalshyDev) [inline src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java]: Move comment below enchant crap and also comment on enchant crap

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java (hunk 2 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java:13]: On all the other classes the null problems were just ignored so I didnt bother, ill add it back

File: src/main/java/dev/j3fftw/litexpansion/Items.java (hunk 4 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/Items.java]: No i just copied sf adding mending to infused elytra

File: src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java (hunk 5 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java]: alright, also do u guys care for javadocs
